ui/public/scripts/services/mcpapi.js

```diff
@@ -200,39 +200,42 @@ angular.module('mobileControlPanelApp').service('mcpApi', [
           return data;
         });
       },
-      integrateService: function(service, integrationKey) {
-        var params = service.integrations[integrationKey];
+      integrateService: function(integration) {
         if (!window.MCP_URL) {
           return new Promise(function(resolve, reject) {
             return reject('No MCP URL');
           });
         }
-        console.log('params: ', params);
+
         let url =
           getMobileServicesURL() +
           '/configure' +
           '/' +
-          params.componentSecret +
+          integration.component +
```

Comment:
I'm seeing `/mobileservice/configure/fh-sync-server/fh-sync-server/mcp-mobile-keys` being called here when it should be `/mobileservice/configure/fh-sync-server/mcp-mobile-keys` I think. I think one of `integration.component` or `integration.target.name` can be removed, not sure which yet